libursa/ZHelpers.cpp

```diff
@@ -1,13 +1,57 @@
 #include "ZHelpers.h"
 
-std::string s_recv(zmq::socket_t &socket) {
+bool s_send_raw(zmq::socket_t *socket, std::string_view payload,
+                int flags = 0) {
+    zmq::message_t message(payload.size());
+    ::memcpy(message.data(), payload.data(), payload.size());
+    return socket->send(message, flags);
+}
+
+bool s_recv_raw(zmq::socket_t *socket, std::string *response) {
```

Comment:
maybe return std::string here and just throw `std::runtime_error` if the recv failed? If there's one thing history taught us it's that people don't check function return codes. In this case response may be uninitialised and C++ UB demons will follow.

If you don't want to do it, consider `[[nodiscard]]`

Reply:
Done